build.gradle.kts

```
@@ -39,7 +39,10 @@ val jsoupVersion = "1.17.1"
 val jjwtVersion = "0.12.3"
 val mockkVersion = "1.13.4"
 val springMockkVersion = "4.0.2"
+val testcontainersVersion = "1.19.3"
+val testcontainersMysqlVersion = "1.19.3"
 val kotestVersion = "5.7.2"
+val kotestExtensionTestcontainers = "2.0.2"
 val kotestExtensionSpring = "1.1.3"
```

저번 PR에서 `kotestExtensionSpring` 변수 뒤에 `version`을 명시해주지 않았군요. 😂
`kotestTestcontainersVersion` 변수명으로 확실히 정해도 좋겠네요!
```suggestion
val kotestTestcontainersVersion = "2.0.2"
val kotestSpringVersion = "1.1.3"
```